native/NativeSampleApps/MobileSyncExplorer/src/com/salesforce/samples/mobilesyncexplorer/ui/MainActivity.java

```diff
@@ -476,6 +529,7 @@ protected FilterResults performFiltering(CharSequence constraint) {
 		@Override
 		protected void publishResults(CharSequence constraint, FilterResults results) {
 			if (results != null && results.values != null) {
+				MainActivity.this.findViewById(R.id.empty).setVisibility(results.count > 0 ? View.GONE : View.VISIBLE);
```

Comment:
`ListViewActivity` used to manage the empty view for us, so now we have that wired up here.